.github/workflows/chart-testing.yaml

```
@@ -35,8 +35,8 @@ jobs:
         id: list-changed
         run: |
           changed=$(ct list-changed --target-branch main)
-          if [[ -n "$changed" ]]; then
-            echo "::set-output name=changed::true"
+          if [[ -n '$changed' ]]; then
+            echo 'changed=true' >> $GITHUB_OUTPUT
           fi
```

This is what is breaking one of the linters.

```suggestion
          if [[ -n "$changed" ]]; then
            echo "changed=true" >> "$GITHUB_OUTPUT"
          fi
```